lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -301,6 +301,33 @@ function dosomething_helpers_format_data($data) {
 }
 
 /**
+ * Utility funciton to get base URL.
+ *
+ * @return string
+ */
+function dosomething_helpers_get_base_url(){
```

Comment:
Yeah, I totally see your point here, @sbsmith86 ! That's what I was thinking too to extract it all into a helper but some variables, like you mentioned, are just a tad bit different. 

I'll change it back to before now!